### DISCUSSION TITLE
Flare Error Count check fails multiple times

### DISCUSSION THREAD

**Discussion by syntafin** [spatie/laravel-health#98] [General]
Good day,

At the moment the `FlareErrorOccurrenceCount` check fails multiple times.

```
cURL error 35: OpenSSL SSL_connect: Connection reset by peer in connection to flareapp.io:443 (see https://curl.haxx.se/libcurl/c/libcurl-errors.html
```

Does someone have this problem too?

**Comment by freekmurze** [comment]
We've seen this happen on older hosts with a outdated version of openSSL that can't handle modern Let's Encrypt certificates. 

If possible, upgrade openSSL on your server.
